content/blog/azure-v6-release/index.md

```diff
@@ -0,0 +1,68 @@
+---
+title: "Pulumi Azure Classic Provider Version 6.0.0"
+date: 2024-09-24T15:04:00
+draft: false
+meta_desc: "Release of the next version of the Azure Classic Pulumi Provider"
+meta_image: meta.png
+authors:
+    - daniel-bradley
+tags:
+    - azure
+    - release
+---
+
+The latest major release of the Azure Classic Pulumi Provider is available now!
+Our 6.0 release contains the [latest upstream changes](https://github.com/hashicorp/terraform-provider-azurerm/releases/tag/v4.0.1) to keep you up-to-date along with the latest features and improvements from Pulumi.
+
+The Pulumi Azure Native provider is our recommended approach for building on Azure which is generated directly from the API specifications so has amazing resource coverage.
+The Azure Classic provider is derived from the upstream Terraform AzureRM provider and can also be used to provision most resources on Azure, and can be used in conjunction with the Native Provider where needed as a useful alternative.
```

Comment:
These two sentences come across as a bit disjointed to me, especially since we just drop into talking about Azure Native immediately after the announcement of the azure classic provider release. A couple ideas on alternatives:

1) Flip the order of these two sentences and add a contrast phase between them ("on the other hand", "alternatively")
2) Keep the order roughly as is, but maybe start the paragraph with something like "The Azure classic provider serves as a complement to our Recommended Azure provider, the Azure Native provider..."

Reply:
Good thoughts. Have re-drafted